chain/cosmos/client.go

```diff
@@ -101,21 +101,6 @@ func (client *Client) Tx(ctx context.Context, txHash pack.Bytes) (account.Tx, pa
 		return &StdTx{}, pack.NewU64(0), fmt.Errorf("parse tx failed: %v", err)
 	}
 
-	// Construct a past context (just before the transaction's height) and query
-	// the sender account to know the nonce (sequence number) with which this
-	// transaction was broadcasted.
-	senderAddr, err := types.AccAddressFromBech32(string(stdTx.From()))
-	if err != nil {
-		return &StdTx{}, pack.NewU64(0), fmt.Errorf("bad address '%v': %v", stdTx.From(), err)
-	}
-	pastContext := client.cliCtx.WithHeight(res.Height - 1)
-	accGetter := auth.NewAccountRetriever(pastContext)
-	acc, err := accGetter.GetAccount(senderAddr)
-	if err != nil {
-		return &StdTx{}, pack.NewU64(0), fmt.Errorf("account query failed: %v", err)
-	}
-	stdTx.signMsg.Sequence = acc.GetSequence()
-
```

Comment:
We still need this part, to attach the tx sequence number to the returned `StdTx`, right?

Reply:
@jazg I think we can keep this snippet before finalising between:
1. How feasible is this on mainnet
2. How feasible is having a constant `0` RenVM nonce for Terra

What do you think?

Reply:
@roynalnaruto Given there's a chance that it can return the wrong sequence number if the user has multiple transactions in the previous block, I think we should leave it out. Also we can definitely require the nonce to be 0 for account based chains, so that's not a problem.